Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_12_1.md

```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### MDE - False Positive Incident Handling
+Fixed typo misspelling in the 'Tag Indicators' task (Changed the 'indicatorsValues' input to "...Allow.Answers.0").
```

Comment:
what's the typo? Seems like you are fixing an issue with tag indicator input.

Reply:
Indeed this is an issue, which I addressed as a typo. Would you like to change that to just as an issue?

Reply:
yes please :)

Reply:
Done!